src/modules/market/components/market-view/market-view.jsx

```diff
@@ -1,25 +1,63 @@
 import React, { Component } from 'react'
+import PropTypes from 'prop-types'
 
 import MarketHeader from 'modules/market/containers/market-header'
 import MarketOutcomesChart from 'modules/market/containers/market-outcomes-chart'
 import MarketOutcomeGraphs from 'modules/market/containers/market-outcome-graphs'
 import MarketOutcomesAndPositions from 'modules/market/containers/market-outcomes-and-positions'
 import MarketTrading from 'modules/market/containers/market-trading'
 
+import parseQuery from 'modules/routes/helpers/parse-query'
+import getValue from 'utils/get-value'
+
 import Styles from 'modules/market/components/market-view/market-view.styles'
 
+import { MARKET_ID_PARAM_NAME } from 'modules/routes/constants/param-names'
+
 export default class MarketView extends Component {
+  static propTypes = {
+    isConnected: PropTypes.bool.isRequired,
+    location: PropTypes.object.isRequired,
+    loadFullMarket: PropTypes.func.isRequired,
+    universe: PropTypes.object.isRequired
+  }
+
+  static loadFullMarket(marketId, props, force) {
```

Comment:
It’s probably better to explicitly consume an options object here and explicitly define the properties from the props object.